README.md

```diff
@@ -0,0 +1,158 @@
+<div align="center">
+
+  <h3><b>Vet_clinic_Schema_Diagram
+</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
```

Comment:
- [ ] Hello, Kindly remove `deployment` header from the table of content as well as in the readme body. This project won't require deployment